v2_1/ws/rest/docs/4_6_conneg.md

```diff
@@ -30,3 +34,19 @@ The list below indicates the valid formats for SDMX RESTful web services, compli
     application/vnd.sdmx.structurespecificmetadata+xml;version=2.1
     application/vnd.sdmx.structure+xml;version=2.1
     application/vnd.sdmx.schema+xml;version=2.1
+    
+### Selection of the Appropriate language
+
+The [Accept-Language header](http://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html) is used to indicate the language preference of the client. Multiple values, along with their respective weights, are possible. For example:
+
+```
+Accept-Language: ru, en-gb;q=0.8, en;q=0.7
+```
+
+### Enabling data compression
+
+Compression should be enabled using the appropriate [Accept-Encoding header](http://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html).
+
+### Helping web caches and Content Delivery Networks (CDN)
+
+The [Vary header](http://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html) is used to indicate the list of headers that are relevant for a particular service. For example, services that offer data in multiple formats will rely on the HTTP Accept header, but this header will likely be irrelevant for services that support only one format. Using the `Vary` header to indicate which headers are effectively used by the server helps web caches and Content Delivery Networks to build appropriate cache keys.
```

Comment:
Just a suggestion to make it more readable: have the first sentence on a lone paragraph and then the example on a different paragraph.

Another example could have to do with the [CORS request](https://developer.mozilla.org/en-US/docs/Web/HTTP/Access_control_CORS#The_HTTP_response_headers) having a response vary based on the Origin header. This is however also related to caching and I don't know if it would add any value by mentionning it here.